.gitignore

```diff
@@ -11,7 +11,6 @@
 *.class
 
 # Generated files
-bin/
```

Comment:
We have this [`CircleCiScripts`](https://github.com/aws-amplify/aws-sdk-android/tree/master/CircleciScripts) dir. Should we re-use that? Possibly and additionally, `git mv CircleCiScripts scripts`; something like that?

Reply:
I like the second suggestion!